### PR TITLE
hometask-1_Grechko_Maria

### PR DESCRIPTION
**Pull request by MariaGrechko**

### CODE REVIEW

**Review by Ikl0 — APPROVED** [review]

File: hometask-1/1.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+sum = 0
+(1...1000).each do |number|
+  number % 3 == 0 ? sum += number : number % 5 == 0 ? sum += number : next
```
Comment (fooooxmr) [inline hometask-1/1.rb:3]: Redo.

File: hometask-1/2.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+LIMIT = 4_000_000
+
+limit_exceed = false
+arr = [1, 2]
+first_num, second_num = arr
+while !limit_exceed do
```
Comment (fooooxmr) [inline hometask-1/2.rb:6]: until limit_exceed

File: hometask-1/4.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+def palindrome?(string)
+  true if string == string.reverse
```
Comment (fooooxmr) [inline hometask-1/4.rb:2]: string == string.reverse

File: hometask-1/4.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+end
+
+def cycle
+  arr = []
+  (100...1000).each do |multiplier_1|
```
Comment (fooooxmr) [inline hometask-1/4.rb:7]: ml_1

**Pull request merged**